docs/src/hackathon/index.md

```diff
@@ -11,40 +11,51 @@ icon: guide
+# Hackathon vom 22.10.2024
 
 ::: tip Zusammenfassung
-- Konferenz: https://audimax.heiconf.uni-heidelberg.de/jxh4-jxx2-tm6c-d37q
+- Konferenz: 
 - [Vorbereitung](#vorbereitung)
 - [Installationshinweise](#installationshinweise)
 :::
 
-Wir laden Sie herzlich zu unserem kommenden Hackathon **am 18.06. von 10:00 Uhr bis 15:30 Uhr** ein. Diese Veranstaltung bietet die Gelegenheit, sich mit den neuesten Updates und Funktionen unserer Komponenten vertraut zu machen.
+Liebe Community,
 
+die Modul 2B Projekte FDPG+ und DSF Community möchten Sie herzlich zu einem Hackathon am 22.10.2024 von 10:00-16:00 Uhr einladen.
+Leiten Sie bitte die Einladung an die technischen Kolleg*innen in den DIZ und den DMSt weiter.
 
-Agenda:
+Fokus des Hackathons liegt auf der Erprobung der neuen Machbarkeit und des neuen Datenselektion- und -Extraktionstools ([TORCH[(https://github.com/medizininformatik-initiative/torch)) im Zusammenspiel mit den DSF Data Sharing Prozessen. Ergänzend wird der [FHIR Data Evaluator](https://github.com/medizininformatik-initiative/fhir-data-evaluator) vorgestellt und Updates zum DSF, dem Feasibility Prozess Plugin und dem Allowlist Management präsentiert.
 
-1. Einführung in Neuerungen (10:00 - 11:00 Uhr):
-   - Begrüßung und Einführung in die neuen Funktionen und Verbesserungen des DSFs
-   - Präsentation der Änderungen der MII DSF-Prozessplugins, sowie von Blaze und Flare 
-2. Updates MII-Komponenten (ab 11:00 Uhr bis verlängerter Pause, um NUM-DIZ-Meeting besuchen zu können):  
-   - DSF
-   - Feasibility
-   - Data Sharing
-   - Data Transfer
-   - KDS Report
-   - Blaze
-   - Flare mit neuer KDS Ontologie
-3. Testen des Prozessplugins "Data Sharing" (ab 14:15 Uhr):
-   - Praktische Anwendung des "Data Sharing"-Prozessplugins auf unserer Testinfrastruktur.
+Wie üblich sollen die Tools und Prozesse hands-on auf der Testinfrastruktur erprobt und deployed werden, dazu folgen weitere Informationen kurz vor dem Hackathon.
 
-Ziel der Veranstaltung ist es die Test- und Produktivsysteme möglichst aller Standorte auf den neusten Stand zu patchen.
+Vorläufiger Zeitplan: Link
 
+10:00-12:30 : Vorstellungen und Updates
+
+•	Vorstellung Datenselektion und Extraktionstool TORCH – FDPG+
+•	Vorstellung FHIR Evaluator – FDPG+
+•	DSF Updates – DSF Community
+•	Feasibility Prozessplugin Updates – DSF Community & FDPG+
+•	Allowlist Management Updates – DSF Community
+
+12:30-14:00 : Mittagspause
+
+14:00-16:00 : Hands-On
+
+•	Erproben der vorgestellten Tools
+•	Datenselektion und Extraktion mit Data Sharing
+•	Support und Fragen zum Deployment und Betrieb
+•	Wir freuen uns auf Ihre Teilnahme!
+
+Viele Grüße,
+Julian Gründner und Maximilian Kurscheidt
+für FDPG+ und DSF Community
 
 
 
 ## Vorbereitung
 In diesem Hackathon ist das Aktualisieren der Systeme ein aktiver Bestandteil und bedarf keiner speziellen Vorbereitung. Für das Testen der Data Sharing Prozesse sollten jedoch einige Voraussetzungen bereits vor dem Termin erfüllt sein.
+###TBD
```

Comment:
?

Reply:
Some things are still missing from FPDG+. But I can remove that.